packages/fxa-payments-server/server/lib/amplitude.js

```diff
@@ -11,12 +11,11 @@ const {
   mapFormFactor,
   mapLocation,
   mapOs,
-  mapUserAgentProperties,
+  mapUserAgentProperties, //eslint-disable-line no-unused-vars
```

Comment:
Sorry about this; I don't understand why the linter didn't run when I had this in the PR.

The better fix is to just delete `mapUserAgentProperties` from this list; if it isn't used, we don't need it.

Reply:
Okay, I'll delete that line.

I noted the reason why the linter passed above in https://github.com/mozilla/fxa/pull/2947#issuecomment-542511065. I was linting `/fxa-shared`, not `/app`

Reply:
Updated the branch